src/main.c

```diff
@@ -62,6 +62,22 @@
 
 #include "pkgcli.h"
 
+#ifndef _HUMANIZE_NUMBER
```

Comment:
Shouldn't we detect this function in `configure.ac` instead of relying on some potentially unportable macro?